okta/resource_user.go

```diff
@@ -84,8 +84,17 @@ func resourceUser() *schema.Resource {
 				Description: "User country code",
 			},
 			"custom_profile_attributes": &schema.Schema{
-				Type:     schema.TypeMap,
-				Optional: true,
+				Type:          schema.TypeMap,
+				Optional:      true,
+				ConflictsWith: []string{"custom_attributes"},
+			},
+			"custom_attributes": &schema.Schema{
```

Comment:
Is there a better approach here? In retrospect I am not sure JSON is the way to go. Perhaps there should be different maps for each type. Although that would require a different map for each array type as well.

Reply:
JSON is definitely going to be the most flexible.

I'm not sure if different maps would even turn out useful -- you'd probably end up having to make a lot of changes over time as folks request more and more things, where as JSON will work for basically everyone.

Reply:
Yea agreed. I wish there were a better approach but this will do.